### PR TITLE
Add keyboard layout configuration example

### PR DESCRIPTION
**Pull request by MDeltaX**

### CODE REVIEW

File: config.in [hunk 1]
```diff
@@ -6,15 +6,19 @@
 
 ### Variables
 #
-# Logo key. Use Mod1 for Alt.
+# Windows key. Use Mod1 for Alt.
```
Comment (emersion) [inline config.in]: Can we keep using "logo"?

File: config.in [hunk 2]
```diff
@@ -6,15 +6,19 @@
...
 set $mod Mod4
+#
```
Comment (emersion) [inline config.in]: Can we not add these?

File: config.in [hunk 3]
```diff
@@ -48,6 +52,24 @@ output * bg @datadir@/backgrounds/sway/Sway_Wallpaper_Blue_1920x1080.png fill
...
+#   Keyboard layout:
+#
+#   input * {
+#       xkb_layout de
+#       xkb_variant neo
+#   }
+#
+#   or:
+#
+#   input * {
```
Comment (emersion) [inline config.in]: One example is probably enough.

File: config.in [hunk 4]
```diff
@@ -6,15 +6,19 @@
...
 set $mod Mod4
+#
 # Home row direction keys, like vim
 set $left h
 set $down j
 set $up k
 set $right l
+#
 # Your preferred terminal emulator
+# set $term kitty
```
Comment (emersion) [inline config.in]: -1

**Comment by ddevault** [comment]
Thank you, but we'll pass on this patch.

**Pull request closed without merging**